src/game.cpp

```diff
@@ -1685,42 +1666,6 @@ bool loadGameInit(const char *fileName)
 	return true;
 }
 
-
-// -----------------------------------------------------------------------------------------
-// Load a file from a save game into the psx.
-// This is divided up into 2 parts ...
-//
-// if it is a level loaded up from CD then UserSaveGame will by false
-// UserSaveGame ... Extra stuff to load after scripts
-bool loadMissionExtras(const char *pGameToLoad, SWORD levelType)
```

Comment:
This was only used in `levels.cpp` and I encountered some circular header dependencies so I thought it is fine to move this there.